frontEnd/src/components/Header.vue

```diff
@@ -77,9 +114,45 @@
 </template>
 
 <script>
+import {getUserByID} from "../services/apiServices"
     export default {
         data: () => ({
             menu: false,
+            notifications:[],
+            name: '',
         }),
+
+        created(){
+          this.getUser(); //call when instance is new
+        },
+        methods: {
+          async getUser(){
```

Comment:
Why use wrapper function for getUserByID? getUserByID is already a wrapper function for the Axios call so I would delete getUser() and just use getUserById()

Reply:
Ah I just copied the formatting from HelloWorld.vue originally to match the style, forgot to change it after though oops

Reply:
I'll do that now